compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -817,6 +817,9 @@ error.invalid.init.invocation=\
 error.invalid.resource.function.invocation=\
   resource function can not be invoked with in a service
 
+error.invalid.function.pointer.invocation.with.type = \
+  cannot invoke function pointer of type 'function'
```

```suggestion
  cannot invoke function pointer of type ''function''
```

---

We should probably say "call" instead of "invoke".
```suggestion
  cannot call function pointer of type ''function''
```

---

Changed.